packages/@react-aria/landmark/src/useLandmark.ts

```diff
@@ -103,9 +103,12 @@ class LandmarkManager {
     this.landmarks.splice(insertPosition, 0, newLandmark);
   }
 
-  public updateLandmark(landmark: Landmark) {
-    this.landmarks = this.landmarks.map((prevLandmark) => prevLandmark.ref.current === landmark.ref.current ? {...prevLandmark, ...landmark} : prevLandmark);
-    this.checkLabels(landmark.role);
+  public updateLandmark(ref: MutableRefObject<HTMLElement>, newFields: Partial<Omit<Landmark, 'ref'>>) {
```

Comment:
I think I liked just passing landmark to this function, that's still what's happening, it's now just two args instead, what was the reason behind this?